SdarotAPI/Models/SeriesInformation.cs

```diff
@@ -2,8 +2,21 @@ namespace SdarotAPI.Models;
 
 public partial class SeriesInformation
 {
-    public string SeriesNameHe { get; set; }
-    public string SeriesNameEn { get; set; }
+    [JsonPropertyName("heb")] public string SeriesNameHe { get; set; } = null;
+    [JsonPropertyName("eng")] public string SeriesNameEn { get; set; } = null;
```

Comment:
`= null` on non-nullable types is meaningless. Use `string.Empty` instead
```suggestion
    [JsonPropertyName("heb")] public string SeriesNameHe { get; set; } = string.Empty;
    [JsonPropertyName("eng")] public string SeriesNameEn { get; set; } = string.Empty;
```